tt_metal/common/base.hpp

```diff
@@ -43,9 +43,9 @@ namespace tt
 enum class TargetDevice : uint8_t
 {
     Model = 0,
-    Versim = 1,
-    Silicon = 2,
-    Golden = 3,
+    Silicon = 1,
+    Golden = 2,
+    VCS = 3,
```

Comment:
?????

Reply:
I don't think we were using anything besides Silicon or Versim, the others seem to be legacy. Maybe we should uplift this to be only Silicon or Simulator

Reply:
Will change `VCS` to `SiliconRemote`. 

So can I remove `TargetDevice::Model` and `TargetDevice::Golden` ?

Reply:
removed